include/NovelRT.h

```diff
@@ -20,9 +20,11 @@
 #include <sstream>
 #include <stdexcept>
 #include <string>
+#include <tuple>
 #include <typeinfo>
 #include <type_traits>
 #include <vector>
+#include <queue>
```

Comment:
nit: order (unless we otherwise have a need)

Reply:
This include isn't actually required. It is being removed.